Too much C++ code breaks for now with this

--- Python-3.7.0/Include/unicodeobject.h~	2018-06-27 03:07:35.000000000 +0000
+++ Python-3.7.0/Include/unicodeobject.h	2018-07-06 16:56:50.335808003 +0000
@@ -1135,7 +1135,7 @@
 */
 
 #ifndef Py_LIMITED_API
-PyAPI_FUNC(const char *) PyUnicode_AsUTF8AndSize(
+PyAPI_FUNC(char *) PyUnicode_AsUTF8AndSize(
     PyObject *unicode,
     Py_ssize_t *size);
 #define _PyUnicode_AsStringAndSize PyUnicode_AsUTF8AndSize
@@ -1162,7 +1162,7 @@
 */
 
 #ifndef Py_LIMITED_API
-PyAPI_FUNC(const char *) PyUnicode_AsUTF8(PyObject *unicode);
+PyAPI_FUNC(char *) PyUnicode_AsUTF8(PyObject *unicode);
 #define _PyUnicode_AsString PyUnicode_AsUTF8
 #endif
 
