--- Python-3.6.3/configure.org	2017-12-16 18:21:53.880800388 +0000
+++ Python-3.6.3/configure	2017-12-16 18:22:19.339799554 +0000
@@ -6937,7 +6937,7 @@
 # tweak BASECFLAGS based on compiler and platform
 case $GCC in
 yes)
-    CFLAGS_NODIST="$CFLAGS_NODIST -std=c99"
+    CFLAGS_NODIST="$CFLAGS_NODIST "
 
     { $as_echo "$as_me:${as_lineno-$LINENO}: checking for -Wextra" >&5
 $as_echo_n "checking for -Wextra... " >&6; }
