--- Python-3.5.0/Python/random.c.org	2015-10-24 09:41:51.981641990 -0400
+++ Python-3.5.0/Python/random.c	2015-10-24 09:43:00.689783020 -0400
@@ -115,7 +115,7 @@
      * need Linux kernel 3.17 or later */
     static int getrandom_works = 1;
     /* Use /dev/urandom, block if the kernel has no entropy */
-    const int flags = 0;
+    const int flags = 1; /* GRND_NONBLOCK */
     int n;
 
     if (!getrandom_works)
